lib/ignored-paths.js

```diff
@@ -92,14 +92,24 @@ class IgnoredPaths {
         }
 
         /**
+         * read ignore filepath
+         * @param {string} filepath, file to add to ig
+         * @returns {array} raw ignore rules
+         */
+        function readIgnoreFile(filepath) {
+            return fs.readFileSync(filepath, "utf8");
+        }
```

Comment:
@not-an-aardvark I am still looking into this. I might have to do something like so. Does that make sense?

```
                addIgnoreFile(this.ig.custom, ignorePath, this.cache);
                addIgnoreFile(this.ig.default, ignorePath, this.cache);
```